### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by aslampr07**

### CODE REVIEW

**Review by lurch — APPROVED** [review]

**Comment by lurch** [comment]
Same typo also appears in the upstream version, if you fancy submitting a PR for that too?

File: usb/device/dev_hid_generic_inout/dev_hid_generic_inout.c [hunk 1]
```diff
@@ -36,7 +36,7 @@
  *
  * There are 2 ways to test the sketch
  * 1. Using nodejs
- *    - Install nodejs and nmp to your PC
+ *    - Install nodejs and npm to your PC
  *    - Install execellent node-hid (https://github.com/node-hid/node-hid) by
```
Comment (lurch) [inline usb/device/dev_hid_generic_inout/dev_hid_generic_inout.c:40]: Haha, also a typo on this line! "execellent" -> "excellent"

**Comment by aslampr07** [comment]
> Same typo also appears in the upstream version, if you fancy submitting a PR for that too?

Sure do, i was checking out the HID example, and noticed this.

**Review by Wren6991 — APPROVED** [review]

**Comment by Wren6991** [comment]
Thanks!

**Pull request merged**